modules/apps/portal-workflow/portal-workflow-api/src/main/java/com/liferay/portal/workflow/portlet/tab/BaseWorkflowPortletTab.java

```diff
@@ -79,6 +89,63 @@ protected Log getLog() {
 		return _logs.get(clazz);
 	}
 
+	protected String getMVCPathAttributeName(String namespace) {
```

Comment:
@MrJohn1911 It seems this method is only used by `getPath`, please inline if so.